TTS/tts/datasets/dataset.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         use_noise_augment: bool = False,
         start_by_longest: bool = False,
         verbose: bool = False,
+        compute_mel_statistics: bool = False,
```

Comment:
The parameter "compute_mel_statistics" is used somewhere? I didn't find the place where this parameter is used.

Reply:
Oh yeah! I added it initially but later decided to move this inside the model as only OverFlow uses it and neural HMM uses it. I will remove it!